java/src/jmri/jmrix/openlcb/swing/stleditor/StlEditorPane.java

```diff
@@ -733,6 +750,10 @@ private void decode(GroupRow groupRow) {
 
                 // Get name
                 if (oper != null) {
+                    if (!name.isEmpty()) {
+                        log.warn("Skipping superfluous input \"{}\" in line \"{}\", missing newline?", token, lines[i]);
```

Comment:
No need to change, but a suggestion. If you change `, missing newline?` to `. Missing newline?`, I think it will be more clear to the user since `Missing newline` is a question to the user, not a comment on what the software is doing. (Dot instead of comma and uppercase letter).